umap/static/umap/js/modules/request.js

```diff
@@ -0,0 +1,157 @@
+// Uses `L._`` from Leaflet.i18n which we cannot import as a module yet
+import { Evented, DomUtil } from '../../vendors/leaflet/leaflet-src.esm.js'
+
+export class HTTPError extends Error {
+  constructor(message) {
+    super(message)
+    this.name = this.constructor.name
+  }
+}
+
+export class NokError extends Error {
```

Comment:
nit: should this be in caps (`NOKError`) ?

Reply:
Documenting what's the difference between `HTTPError` and `NOKError` could be useful :-)